lua/sfm/actions.lua

```diff
@@ -756,8 +800,19 @@ function M.move()
   end)
 end
 
---- copy file/directory
+--- copy file/s directory/ies
 function M.copy()
+  local selections = M._ctx:get_selections()
+  local count = vim.tbl_count(selections)
+  if count > 1 then
+    M._copy_selections()
+  else
+    M.copy_single()
+  end
+end
+
+--- copy a single file/directory
+function M.copy_single()
```

Comment:
The method confused me. How about changing it to copy_current? What do you think? (do the same to other method *_single)

Reply:
that makes more sense, done